src/Stryker.Core/Stryker.Core/Stryker.Core.csproj

```diff
@@ -11,8 +11,8 @@
     <StartupObject />
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
     <VersionPrefix>0.9.0</VersionPrefix>
-    <PackageId>StrykerMutator.Core</PackageId>
-    <Description>The core package for Stryker.NET. Used by other Stryker packages to run. Please install a Stryker.Runner.* package to use this package.</Description>
+    <PackageId>stryker</PackageId>
+    <Description>The core package for Stryker.NET. Used by other Stryker packages to run. Install dotnet-stryker to run stryker from the commandline.</Description>
```

Comment:
I think we should make it more clear that this package is NOT the CLI package and this tool cannot be run as a CLI tool. I guess a lot of people will download this by mistake, instead of `dotnet-stryker`